core/MusicControl/src/main/java/org/jjazz/musiccontrol/api/MusicGenerationQueue.java

```diff
@@ -200,12 +200,6 @@ public void run()
 
             if (incoming != null)
             {
```

Comment:
Please leave the comments in this case. There have been a few complex bugs with this thread so it's convenient for me to just uncomment. I can not use FINE for performance reasons here.